backend/main_page/views/user_popular_topic_view.py

```diff
@@ -0,0 +1,98 @@
+
+from django.views import View
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.renderers import JSONRenderer
+import requests
+
+# Constants for the RapidAPI Instagram Reels feed API
+from rest_framework.views import APIView
+
+RAPIDAPI_KEY = "e619c3e4afmsh92c8cabd68cf43cp1bebaejsn15df63d545be"
+RAPIDAPI_HOST = "instagram-data1.p.rapidapi.com"
```

Comment:
Please move them to .env 😭 they are not meant to be shared with the whole internet..